code/game/objects/items/weapons/implant/implant.dm

```diff
@@ -29,6 +29,7 @@
 
 //return TRUE for implanter icon update.
 /obj/item/weapon/implant/proc/install(var/mob/living/target, var/organ, var/mob/user)
+	world << "Implant installing"
```

Comment:
These look like leftover debug messages? Did you mean to leave these here?

Reply:
i did not, good catch